### ISSUE TITLE
Module namespace is not the same the module name

### ISSUE THREAD

**Issue by nammaianh** [pingpong-labs/modules#144]
I have my module named INewModule. Every class created using Artisan console has the the namespace as Inewmodule instead of INewModule. It makes me tired of editing the namespace, and sometimes got class-not-found exception. :sob:

Change this, please!!! :sob:

**Comment by gravitano** [comment]
Hi @nammaianh,

I am sorry, the class namespace is using `StudlyCase` convention. This is better, because we follow `PSR-4` autoloading standard.

**Comment by nammaianh** [comment]
Yes, StudlyCase. But it is Inewmodule (only the first character is uppercase), not INewModule (All upppercase characters are keep uppercase) :|

**Comment by thanhdble** [comment]
same issue
